src/smt/optimization_solver.cpp

```diff
@@ -31,18 +31,26 @@ OptimizationSolver::OptimizationSolver(SmtEngine* parent)
       d_optChecker(),
       d_objectives(),
       d_results(),
-      d_objectiveCombination(BOX)
+      d_objectiveCombination(LEXICOGRAPHIC)
```

Comment:
Why is `LEXICOGRAPHIC` the default mode? Please either document this fact in the `h` file, or add another mode `UNSET`, and then assert that the mode is not `UNSET` where you actually compute the optimized values. Otherwise the user may be unaware of this decision.

Reply:
Lexico is more intuitive than Box. I will add some comment on it.

Reply:
commented in checkOpt